app/models/katello/concerns/content_aspect_host_extensions.rb

```diff
@@ -0,0 +1,22 @@
+module Katello
+  module Concerns
+    module ContentAspectHostExtensions
+      extend ActiveSupport::Concern
+
+      included do
+        has_one :content_aspect, :class_name => '::Katello::Host::ContentAspect', :foreign_key => :host_id, :inverse_of => :host, :dependent =>  :destroy
+        has_one :content_view, :through => :content_aspect
+        has_one :lifecycle_environment, :through => :content_aspect
```

Comment:
it isn't, was more there for convenience.  so you can do:

host.content_view
rather than:
host.content_aspect.try(:content_view) 

when trying to fetch the content_view for a host.  You didn't seem to like that above.  Any thoughts?